lib/spack/spack/test/database.py

```diff
@@ -111,9 +111,6 @@ def test_default_queries(database):
     libraries = spec['zmpi'].libs
     assert len(libraries) == 1
 
-    cppflags_expected = '-I' + spec.prefix.include
-    assert spec['zmpi'].cppflags == cppflags_expected
```

Comment:
@adamjstewart I would rather modify the mocking if it needs to look "more real" than removing a test.